src/main/telemetry/smartport.h

```diff
@@ -27,5 +28,6 @@ bool initSmartPortTelemetryExternal(smartPortWriteFrameFn *smartPortWriteFrameEx
 void handleSmartPortTelemetry(void);
 void processSmartPortTelemetry(smartPortPayload_t *payload, volatile bool *hasRequest, const uint32_t *requestTimeout);
 
-void smartPortWriteFrameSerial(const smartPortPayload_t *payload, serialPort_t *port, uint16_t checksum);
-void smartPortSendByte(uint8_t c, uint16_t *checksum, serialPort_t *port);
+struct serialPort_s;
```

Comment:
@martinbudden: Just out of interest, what is the reason for declaring the struct here being preferable over including `drivers/serial.h`? The latter seems to be somehow 'cleaner' to me.